config/env/test.json

```diff
@@ -5,6 +5,10 @@
   "readyRetryIntervalMS": 10000,
   "requireAuth": false,
   "schedulerIntervalMS": 10000,
+  "carStorage": {
+    "mode": "s3",
+    "s3BucketName": "myS3Bucket"
```

Comment:
is there a better default to use here?  Maybe we should put whatever we are actually using in our infra?  @smrz2001

Reply:
Yeah, we use `ceramic-${env}-cas` so we could use `ceramic-dev-cas` here.